code/workspaces/infrastructure-api/src/controllers/controllerHelper.js

```diff
@@ -7,7 +7,7 @@ function validateAndExecute(request, response, errorMessage, execute) {
 
   if (!errors.isEmpty()) {
     logger.error(`${errorMessage}: ${JSON.stringify(request.body)}`);
-    logger.error(errors.mapped());
+    logger.error(`${JSON.stringify(errors.mapped())}`);
```

Comment:
How come this needs to be in a template string rather than just `JSON.stringify`?

Reply:
Fixed!